symengine/constants.h

```diff
@@ -69,6 +70,9 @@ extern SYMENGINE_EXPORT RCP<const Constant> EulerGamma;
 extern SYMENGINE_EXPORT RCP<const Infty> Inf;
 extern SYMENGINE_EXPORT RCP<const Infty> NegInf;
 extern SYMENGINE_EXPORT RCP<const Infty> ComplexInf;
+
+// Not a Number
+extern SYMENGINE_EXPORT RCP<const NaN> Nan;
```

Comment:
`nan` is a builtin as well. Sorry about that. Can you change this back to `Nan`?